.env.sample

```diff
@@ -0,0 +1,2 @@
+GITHUB_TOKEN=
```

Comment:
It's critical to avoid hardcoding API tokens in the source code. Instead, consider using environment variables or a secrets management service to keep sensitive information secure.